modules/ROOT/pages/database-administration/composite-databases/querying-composite-databases.adoc

```diff
@@ -17,7 +17,40 @@ Queries submitted to a Composite database may contain several <<query-use, `USE`
 
 Each constituent graph is named after the alias that introduces it into the Composite database.
 
-The examples assume the same setup as the one created in xref:composite-databases/administration.adoc[].
+[[composite-databases-queries-graph-set-up]]
+== Graph set-up
+
+The following set-up is required to recreate the below examples:
+
+.Create a standard database `movies2022`
+[source, cypher]
+----
+CREATE DATABASE movies2022
+----
+
+.Create a composite database `cineasts``
+[source, cypher]
+----
+CREATE COMPOSITE DATABASE cineasts
+----
+
+.Create alias `cineasts`.`latest` on a composite database
```

Comment:
I think we have tried to always call them `database aliases` in documentation (there might be some exceptions like referring to a recently specified database alias as just alias but in general at least), might be good to go through properly and confirm that it still holds and that this move has not broken that.

Might also be nice to mention that this alias is for a local database explicitly. And I think it might sound better to create the alias in the composite and not on the composite so as to not think you're making an alias for the composite?

```suggestion
.Create database alias `cineasts`.`latest` for a local database in a composite database
```
or
```suggestion
.Create local database alias `cineasts`.`latest` in a composite database
```

Reply:
Good suggestions (I think 'for a local database' is the most accurate?)

Reply:
I think both are accurate as we do generally refer to database aliases for local databases as local database aliases but it might read better to say 'for a local database' in this case